openwisp_monitoring/settings.py

```diff
@@ -3,5 +3,14 @@
 AUTO_GRAPHS = getattr(
     settings,
     'OPENWISP_MONITORING_AUTO_GRAPHS',
-    ('traffic', 'wifi_clients', 'uptime', 'packet_loss', 'rtt',),
+    (
+        'traffic',
+        'wifi_clients',
+        'uptime',
+        'packet_loss',
+        'rtt',
+        'memory',
+        'load',
```

Comment:
what do you guys think of renaming this and other occurrences of load (when using internally as metric name, chart name, metric key) to `cpu`? since we have `memory` and `disk`, `load` feels a bit out of place.

I just had to rename the data from influxdb and while looking for it I couldn't find it:

```
> show measurements
name: measurements
name
----
br_lan
br_wan
device_data
disk
eth0
eth0_1
eth0_2
load
memory
ping
tap0
wlan0
wlan1
```
After a while I realized I was looking for `cpu` but it's named `load` from load aveage, but I believe the value we're storing and showing in the "CPU Load" chart doesn't have much to do with load averages anymore and it would be better to refer to it as `cpu`.

@PabloCastellano @nepython

Reply:
I agree. Indeed I opened an issue to review measurements names https://github.com/openwisp/openwisp-monitoring/issues/111